cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -672,6 +690,25 @@ sendEKGDirectDouble ekgDirect name val = do
         Label.set label (Text.pack (show val))
         pure $ Map.insert name label registeredMap
 
+sendEKGDirectPrometheusLabel :: EKGDirect -> Text -> [(Text,Text)] -> IO ()
+sendEKGDirectPrometheusLabel ekgDirect name labels = do
+  modifyMVar_ (ekgLabels ekgDirect) $ \registeredMap -> do
+    case Map.lookup name registeredMap of
+      Just label -> do
+        Label.set label (presentPrometheusM labels)
+        pure registeredMap
+      Nothing -> do
+        label <- EKG.getLabel name (ekgServer ekgDirect)
+        Label.set label (presentPrometheusM labels)
+        pure $ Map.insert name label registeredMap
+  where
+    presentPrometheusM :: [(Text, Text)] -> Text
+    presentPrometheusM =
+      label . map pair
+      where
+        label pairs = "{" <> Text.intercalate "," pairs <> "} 1"
```

Comment:
please double-check Prometheus page to not contain `... 1 1`

Reply:
You were right